neo/SmartContract/InteropService.NEO.cs

```diff
@@ -243,57 +243,56 @@ private static bool Account_IsStandard(ApplicationEngine engine)
 
         private static bool Contract_Create(ApplicationEngine engine)
         {
-            byte[] script = engine.CurrentContext.EvaluationStack.Pop().GetByteArray();
-            if (script.Length > 1024 * 1024) return false;
+            byte[] binary = engine.CurrentContext.EvaluationStack.Pop().GetByteArray();
```

Comment:
I think the parameter of `Contract.Create()` should be a script that has removed the header. Header checking should be done in the client side.

Reply:
So Script should be use only for neo-cli, right?

Reply:
`ScriptFile`. Yes. Or we can rename it to `NefFile`.